MANIFEST.in

```diff
@@ -1,9 +1,10 @@
 include *.txt
 recursive-include wof/flask/templates *
-recursive-include wof/examples/flask/odm2/timeseries *
+recursive-include wof/examples *
```

Comment:
@lsetiawan and @emiliom maybe this is easier to grasp as it is more "generic." Here we include everything from `wof/examples` and then `prune` what we don't want.